### DISCUSSION TITLE
The Compose file './docker-compose.yaml' is invalid

### DISCUSSION THREAD

**Discussion by mohmafia** [mediacms-io/mediacms#297] [Q&A]
Cloning into 'mediacms'...
remote: Enumerating objects: 2034, done.
remote: Counting objects: 100% (1939/1939), done.
remote: Compressing objects: 100% (1176/1176), done.
remote: Total 2034 (delta 849), reused 1657 (delta 713), pack-reused 95
Receiving objects: 100% (2034/2034), 12.64 MiB | 15.12 MiB/s, done.
Resolving deltas: 100% (852/852), done.
root@vmi494604:/# cd mediacms
root@vmi494604:/mediacms# docker-compose up
ERROR: The Compose file './docker-compose.yaml' is invalid because:
services.migrations.depends_on contains an invalid type, it should be an array

**Comment by mgogoulos** [comment]
Hi, this indicates that you are using an older version of docker-compose

Can you check what is the output of  `docker-compose -v`?
You have to upgrade to a new version, eg 1.29.2, check the docker + docker compose instructions on 

https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#3-docker-installation

**Reply by mohmafia** [reply]
ok now i get this....
celery_worker_1  | 2021-10-16 01:09:12,875 INFO success: celery_short entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
celery_worker_1  | 2021-10-16 01:09:12,876 INFO success: celery_long entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
celery_worker_1  | celery multi v4.4.7 (cliffs)
celery_worker_1  | > Starting nodes...
celery_worker_1  |      > short1@5e416828db46: celery multi v4.4.7 (cliffs)
celery_worker_1  | > Starting nodes...
migrations_1     | Running loaddata and creating admin user
web_1            | WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x559cd04e3fd0 pid: 55 (default app)
web_1            | *** uWSGI is running in multiple interpreter mode ***
web_1            | spawned uWSGI master process (pid: 55)
web_1            | spawned uWSGI worker 1 (pid: 62, cores: 2)
web_1            | spawned uWSGI worker 2 (pid: 63, cores: 2)
web_1            | running "unix_signal:15 gracefully_kill_them_all" (master-start)...
celery_worker_1  |      > long1@5e416828db46: OK
celery_worker_1  |      > short2@5e416828db46: 2021-10-16 01:09:14,573 INFO reaped unknown pid 62
celery_worker_1  | OK
celery_worker_1  | 2021-10-16 01:09:14,731 INFO reaped unknown pid 65
celery_worker_1  | 2021-10-16 01:09:14,786 INFO exited: celery_long (exit status 0; expected)
celery_worker_1  | OK
celery_worker_1  | 2021-10-16 01:09:16,006 INFO reaped unknown pid 68
celery_worker_1  | 2021-10-16 01:09:16,072 INFO exited: celery_short (exit status 0; expected)
migrations_1     | Installed 22 object(s) from 1 fixture(s)
migrations_1     | Installed 6 object(s) from 1 fixture(s)
migrations_1     | Superuser created successfully.
migrations_1     | Created admin user with password: somepassword
migrations_1     | RUNNING COLLECTSTATIC
migrations_1     |
migrations_1     | 1453 static files copied to '/home/mediacms.io/mediacms/static'.
mediacms_migrations_1 exited with code 0

root@vmi:~# docker -v
Docker version 20.10.9, build c2ea9bc
root@vmi:~# docker-compose -v
docker-compose version 1.29.2, build 5becea4c
